docs/field-type-validator-usage.md

```diff
@@ -0,0 +1,73 @@
+# Field Type Validator
+
+Field Type Validator is a base class to validate inputs in field type.
+
+## How to use it?
+
+```javascript
+class Validator extends global.eZ.BaseFieldValidator {
```

Comment:
Pointing to `global` object might be a bit confusing unless you provide a full example:
```javascript
(function (global) {
    class Validator extends global.eZ.BaseFieldValidator { ... }
})(window);
```